com.unicornsonlsd.finamp.yaml

```diff
@@ -0,0 +1,298 @@
+app-id: com.unicornsonlsd.finamp
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: finamp
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0:ro
+  - --share=network
+modules:
+  - name: finamp
+    buildsystem: simple
+    build-commands:
+      - cp -r bundle/finamp /app/finamp
+      - cp -a bundle/lib/. /app/lib/
+      - cp -a bundle/data/. /app/data/
+      - chmod +x /app/finamp
+      - install -Dm644 icons/16x16/apps/finamp.png /app/share/icons/hicolor/16x16/apps/com.unicornsonlsd.finamp.png
+      - install -Dm644 icons/32x32/apps/finamp.png /app/share/icons/hicolor/32x32/apps/com.unicornsonlsd.finamp.png
+      - install -Dm644 icons/48x48/apps/finamp.png /app/share/icons/hicolor/48x48/apps/com.unicornsonlsd.finamp.png
+      - install -Dm644 icons/64x64/apps/finamp.png /app/share/icons/hicolor/64x64/apps/com.unicornsonlsd.finamp.png
+      - install -Dm644 icons/128x128/apps/finamp.png /app/share/icons/hicolor/128x128/apps/com.unicornsonlsd.finamp.png
+      - install -Dm644 icons/256x256/apps/finamp.png /app/share/icons/hicolor/256x256/apps/com.unicornsonlsd.finamp.png
+      - install -Dm644 icons/384x384/apps/finamp.png /app/share/icons/hicolor/384x384/apps/com.unicornsonlsd.finamp.png
+      - install -Dm644 icons/512x512/apps/finamp.png /app/share/icons/hicolor/512x512/apps/com.unicornsonlsd.finamp.png
+      - mkdir -p /app/bin
+      - ln -s /app/finamp /app/bin/finamp
+      - sed -i 's/Icon=finamp/Icon=com.unicornsonlsd.finamp/' finamp.desktop.m4
```

Comment:
the proper way is to use the `desktop-file-edit` command

the better way is that you just keep the orignal name when installing and put `rename-icon` in the manifest. It handle fixing the .desktop file.

Reply:
I'm currently getting 

```
Errors were raised during this compose run:
com.unicornsonlsd.finamp
  E: gui-app-without-icon

general
  E: filters-but-no-output
```

Do you know what I'm doing wrong?

Reply:
The linter should have a specific error message for that, let's see. It can be one of 3-4 things